Dockerfile

```diff
@@ -1,28 +1,29 @@
-FROM debian:jessie
-MAINTAINER Erik Dasque <erik@frenchguys.com>
+FROM debian
 
-RUN apt-get update && apt-get install --no-install-recommends -y build-essential \
-    gcc \
-    unzip \
-    wget \
-    zip \
-    gcc-avr \
-    binutils-avr \
+RUN apt-get update && apt-get install --no-install-recommends -y \
     avr-libc \
+    avrdude \
+    binutils-arm-none-eabi \
+    binutils-avr \
+    build-essential \
     dfu-programmer \
     dfu-util \
+    gcc \
     gcc-arm-none-eabi \
-    binutils-arm-none-eabi \
-    libnewlib-arm-none-eabi \
+    gcc-avr \
     git \
+    libnewlib-arm-none-eabi \
     software-properties-common \
-    avrdude \
+    unzip \
+    wget \
+    zip \
     && rm -rf /var/lib/apt/lists/*
 
-ENV keyboard=ergodox
-ENV subproject=ez
-ENV keymap=default
+ENV KEYBOARD=ergodox_ez
+ENV KEYMAP=default
+
+VOLUME /qmk_firmware
+WORKDIR /qmk_firmware
+COPY . .
 
-VOLUME /qmk
-WORKDIR /qmk
-CMD make clean ; make keyboard=${keyboard} subproject=${subproject} keymap=${keymap}
+CMD make $KEYBOARD:$KEYMAP
```

Comment:
I think that expressing the cmd arguments as an array of strings is somewhat standard on dockerfiles. I don't care at all, I'm just commenting it.

Reply:
The format in this case actually depends on the `Makefile` since we're calling `make`. The old build system that used subprojects took arguments as kwargs, but that's since been deprecated. The new format is `keyboard:keymap:target` (where `keymap` defaults to `default` and `target` defaults to no target).